plugins/action_send_notification/src/sendNotification.ts

```diff
@@ -102,11 +132,7 @@ const sendNotification: ActionPluginType = async ({ parameters, applicationData,
             (!rejected || rejected.length === 0) &&
             (!pending || pending.length === 0)
           ) {
-            console.log(
-              `Email successfully sent to: ${envelope.to}\ncc:${envelope.cc || ''}\nbcc: ${
-                envelope.bcc || ''
-              }\nSubject: ${subject || ''}\n`
-            )
+            console.log(`Email successfully sent to: ${envelope.to}\n`)
```

Comment:
Note the difference here. The earlier "Sending to..." log should show the recipients who the email *would* go to in normal "live" operation, whereas this value shows who it was *actually* sent to based on the operation mode. So compare these values in the console to make sure its behaving as it's intended.